### DISCUSSION TITLE
[Feature] Enhancing Update Synchronization Between Mobile and Web Apps

### DISCUSSION THREAD

**Discussion by jhrcc** [immich-app/immich#14522] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I hope that updates and downloads for the mobile app can be synchronized with the web version. This way, after updating the server, users can simply open the mobile app for immediate updates and downloads. For minor version updates, I would like to suggest implementing hot updates via a data package approach, making the process much more convenient.I hope that the mobile app's updates and downloads can be synchronized with the web version. This way, after updating the server, users can simply open the mobile app to access immediate updates and downloads. For minor version updates, I would suggest implementing hot updates through a data package approach, as this would significantly enhance convenience and streamline the user experience.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Unless noted in the release notes, there's no need for the app and server to be perfectly in sync.
